src/store/poll/utilities.js

```diff
@@ -0,0 +1,52 @@
+import { database } from '../../../firebase'
+
+const generateRandomNum = () =>
+  Math.floor(Math.random() * 4) + 1
+
+export const generateHash = () =>
```

Comment:
Just a note on naming - what we're generating is not really technically a "hash" as much as a "key" of sorts. To "hash" something you need an input that creates an output. If it's not too much effort, I'd highly consider renaming

Reply:
✔️